src/components/Registration/BusinessAddresses.vue

```diff
@@ -4,25 +4,17 @@
     <template v-if="!isEditing">
       <v-row no-gutters id="address-summary">
         <v-col md="3" class="mr-n1"><label><strong>Business Addresses</strong></label></v-col>
-        <v-col md="4">
+        <v-col md="4" class="pr-4">
```

Comment:
This is to have a 20px "gutter" between the mailing and delivery addresses.

I added this same style to the delivery address below so they are both the same width.